plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAnalyze.java

```diff
@@ -765,11 +778,212 @@ public void testCollectStatsAfterColumnAdded(boolean collectOnWrite)
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testNoStats()
+            throws IOException, URISyntaxException
+    {
+        String tableName = copyResourcesAndRegisterTable("no_stats");
+        assertQuery(
+                "SELECT * FROM " + tableName,
+                """
+                        VALUES
+                        (42, 'foo'),
+                        (12, 'ab'),
+                        (null, null),
+                        (15, 'cd'),
+                        (15, 'bar')
+                        """);
+
+        assertUpdate("ANALYZE " + tableName, 5);
+        assertQuery(
+                "SHOW STATS FOR " + tableName,
+                """
+                        VALUES
+                        ('c_int', null, 3.0, 0.2, null, 12, 42),
+                        ('c_str', 10.0, 4.0, 0.2, null, null, null),
+                        (null, null, null, null, 5.0, null, null)
+                        """);
+
+        deleteRecursively(Path.of(new URI(getTableLocation(tableName)).getPath()), ALLOW_INSECURE);
```

Comment:
Let's run the assertion on the select here again too. Prove that the analyze didn't change the results